### DISCUSSION TITLE
[Question] aXenServerList_groupHeader

### DISCUSSION THREAD

**Discussion by KuroRaiken** [aXenDeveloper/ips-app-advanced-serverlist#53] [Ideas]
Why was this ipsDataItem aXenServerList_groupHeader code added?
As far as I understand, people asked to make it so that inside 1 server there were sub-servers that were shown when you click on the server.
This is necessary for a large number of servers, mainly CS)

 In the screenshot, an example of an idea how to implement or make some kind of slider for servers)
![165135138-6485bf8d-812a-46b1-81a7-286dd10cb33b](https://user-images.githubusercontent.com/103626579/166093965-04941b85-7dc6-4284-b175-099252d7ba39.png)

**Comment by aXenDeveloper** [comment]
Are you suggesting that I shouldn't develop a FREE app?

**Reply by KuroRaiken** [reply]
> Вы предлагаете мне не разрабатывать БЕСПЛАТНОЕ приложение?

If the current make some functions paid)
